include/kernel.h

```diff
@@ -1360,7 +1371,7 @@ static inline s64_t __ticks_to_ms(s64_t ticks)
 #ifdef _NEED_PRECISE_TICK_MS_CONVERSION
 	/* use 64-bit math to keep precision */
 	return (u64_t)ticks * sys_clock_hw_cycles_per_tick * MSEC_PER_SEC /
-		CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC;
```

Comment:
Here, i agree with using the "variable" instead of the macro name, cause it is a run-time expression.